tests/e2e-pw/playwright.config.ts

```diff
@@ -23,7 +23,7 @@ const config: ExtendedPlaywrightTestConfig = {
 		: 90 * 1000,
 	expect: { timeout: 20 * 1000 },
 	outputDir: './test-results/report',
-	globalSetup: require.resolve( './global-setup' ),
+	// globalSetup: require.resolve( './global-setup' ),
```

Comment:
Currently, `global-setup` is registering sessions both for [admins](https://github.com/woocommerce/woocommerce-blocks/blob/159185ac89e32ea8ae2d05b19552626371848297/tests/e2e-pw/global-setup.ts#L60-L91) and [shoppers](https://github.com/woocommerce/woocommerce-blocks/blob/159185ac89e32ea8ae2d05b19552626371848297/tests/e2e-pw/global-setup.ts#L101-L135). Do you have an alternative approach for that in mind? I'm asking as @opr created utils based on that in https://github.com/woocommerce/woocommerce-blocks/blob/add/cart-pw-tests/tests/e2e-pw/utils/user-state.tsx. 

Given that this is a draft PR, I also might have caught the wrong moment to comment on this. 🙈

Reply:
We might not end up using mine if we can figure out how to get the Gutenberg utils in, we we will have the utils passed into `test` that we can use instead.

Reply:
@opr Thanks for your reply! 

@nielslange As @opr said, the Gutenberg utils package does the login automatically.

Reply:
That's great to hear, @gigitux.